chain/jsonrpc/res/debug.html

```diff
@@ -3,6 +3,22 @@
 <head>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script>
+        function convertTime(total_seconds) {
+            let hours = Math.floor(total_seconds / 3600)
+            let minutes = Math.floor((total_seconds - (hours * 3600)) / 60)
+            let seconds = total_seconds - (hours * 3600) - (minutes * 60)
+            if (hours > 0) {
+                if (minutes > 0) {
+                    return `${hours}h ${minutes}m ${seconds}s`
+                } else {
+                    return `${hours}h ${seconds}s`
+                }
+            }
+            if (minutes > 0) {
+                return `${minutes}m ${seconds}s`
+            }
+            return `${seconds}s`
```

Comment:
Nodes can run for days so perhaps also spell out days?  How about:

```suggestion
            const seconds = total_seconds % 60;
            const total_minutes = (total_seconds - seconds) / 60;
            const minutes = total_minutes % 60;
            const total_hours = (total_minutes - minutes) / 60;
            const hours = total_hours % 24;
            const days = (total_hours - hours) / 24;
            if (days > 0) {
                return `${days}d ${hours}:${minutes}`
            } else {
                return `${hours}:${minutes}:${seconds}`
            }
```

Reply:
Thanks.